AppleScript/AppleScript.sublime-syntax

```diff
@@ -1021,15 +1134,19 @@ variables:
     )\b
   preposition: |-
     \b(?x:
-      above | against | apart\s+from | around | aside\s+from | at | below
-    | beneath | beside | between | by | for | from | instead\s+of | into | on
-    | onto | out\s+of | over | thru | under
+      above | against | apart\s+from | around | aside\s+from | at
+    | below | beneath | beside | between | by
+    | for | from
+    | instead\s+of | into
+    | on | onto | out\s+of | over
+    | thru
+    | under
     )\b
   # https://developer.apple.com/library/archive/documentation/AppleScript/Conceptual/AppleScriptLangGuide/reference/ASLR_classes.html#//apple_ref/doc/uid/TP40000983-CH1g-SW8
   unit_type: |-
     \b(?x:
       # Optional length dimensions (cubic cannot be miles/km)
-      (?:(?: square | cubic (?! \s+ kilomet | miles )) \s+)?
+      (?:(?: square | cubic (?!\s+kilomet | miles )) \s+)?
```

Comment:
I realize I have a bug here, not `\s+`ing before the `miles` exception to cubic units.